File: apzkr-pzpi-21-2-volnykov-kyrylo/Task1-Server/AquaTrack/AquaTrack/Models/Aquarium.cs
namespace AquaTrack.Models
{
    public class Aquarium
    {
        public int AquariumId { get; set; }
        public int UserId { get; set; }
        public string AquariumType { get; set; }
        public float? Acidity { get; set; }
        public float? WaterLevel { get; set; }
        public float? Temperature { get; set; }
        public float? Lighting { get; set; }

        public User User { get; set; }
        public List<Inhabitant> Inhabitants { get; set; }
        public List<FeedingSchedule> FeedingSchedules { get; set; }
    }
}

File: apzkr-pzpi-21-2-volnykov-kyrylo/Task4-MobileClient/AquaTrackMobile/app/src/main/java/com/example/aquatrackmobile/models/Aquarium/Aquarium.kt
package com.example.aquatrackmobile.models.Aquarium

import com.example.aquatrackmobile.models.FeedingSchedule.FeedingSchedule
import kotlinx.serialization.Serializable

@Serializable
data class AquariumResponse(
    val aquarium: Aquarium
)

@Serializable
data class Aquarium(
    val aquariumId: Int,
    val aquariumType: String,
    val acidity: Double,
    val waterLevel: Double,
    val temperature: Double,
    val lighting: String
)